content.md

```diff
@@ -10,17 +10,17 @@ Email is crucial for professional communication. Checking it frequently is essen
 
 #### Choose an Email Client
 
-- **Desktop Clients**: Examples include [Microsoft Outlook](https://www.microsoft.com/en-us/microsoft-365/outlook/email-and-calendar-software-microsoft-outlook), [Apple Mail](https://apps.apple.com/us/app/mail/id1108187098), and [Thunderbird](https://www.thunderbird.net/en-US/). For DPI, **we recommend using Outlook**.
+- **Desktop Clients**: Examples include [Microsoft Outlook](https://www.microsoft.com/en-us/microsoft-365/outlook/email-and-calendar-software-microsoft-outlook), [Apple Mail](https://apps.apple.com/us/app/mail/id1108187098), and [Thunderbird](https://www.thunderbird.net/en-US/).
 
 ![](assets/mac-outlook.png) ![](assets/mac-email.png)
 
-- **Web Clients**: For student emails we use [webmail.illinois.edu](https://webmail.illinois.edu), however other common examples include [outlook.office.com](outlook.office.com), [mail.google.com](mail.google.com) and [mail.yahoo.com](mail.yahoo.com). This is dependent on whichever email service you are using.
```

Comment:
Keep this

Reply:
But students don't have illinois.edu emails anymore, right?

Reply:
ah good point.